### PR TITLE
Remove "-pre" from the version

### PR DESCRIPTION
**Pull request by maturar**
Issues: #

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [x] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,17 +1,10 @@
...
 
-members = [
-    "api",
-    "common",
-    "agent",
-    "server",
-    "grpc",
-    "ank"
-    ]
+members = ["api", "common", "agent", "server", "grpc", "ank"]
```
Comment (krucod3) [inline Cargo.toml:4]: Not that it really matters that much, but do we need this change?

Comment (krucod3) [reply Cargo.toml:4]:
```suggestion
members = [
    "api", 
    "common", 
    "agent", 
    "server", 
    "grpc", 
    "ank"
    ]
```

**Review by krucod3 — APPROVED** [review]
LGTM 👍

File: Cargo.toml (hunk 1 above)
Comment (maturar) [reply Cargo.toml]: We do not need this change. This is done by the autoformatting tool.

**Pull request merged**